src/client/.eslintrc.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
 		"browser": true
 	},
 	"rules": {
-		"no-console": "off",
+		"no-console": 1,
```

Comment:
This change has nothing to do with the admin panel and should be a separate PR.

Reply:
See PR #237